src/masonite/drivers/mail/BaseMailDriver.py

```diff
@@ -40,6 +43,42 @@ def mail_from_header(self):
     def mail_to_header(self):
         return ','.join(self.to_addresses)
 
+    def text(self, content):
+        """Set the text content of the email.
+
+        Arguments:
+            content {string} -- The email text content.
+
+        Returns:
+            self
+        """
+        self.text_content = content
+        return self
+
+    def html(self, content):
+        """Set the html content of the email.
+
+        Arguments:
+            content {string} -- The email html content.
+
+        Returns:
+            self
+        """
+        self.html_content = content
+        return self
+
+    @property
+    @deprecated('Please use `.text_content` and `.html_content` instead.')
+    def message_body(self):
+        """Returns the body of the message
+        """
+        return self.html_content or self.text_content
+
+    @message_body.setter
+    @deprecated('Please use `.text_content` and `.html_content` instead.')
```

Comment:
I think we can do this without deprecating it. Just keep the setter property as message_body and we can detect the mimetype by a new property like `_mimetype`